### PR TITLE
Add more log for KeyVaultKeyStore.

### PR DESCRIPTION
**Pull request by chenrujun**
As title.

### CODE REVIEW

**Review by backwind1233 — APPROVED** [review]
LGTM

File: sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/KeyVaultKeyStore.java [hunk 1]
```diff
@@ -131,11 +132,16 @@ public KeyVaultKeyStore() {
                     .map(Boolean::parseBoolean)
                     .orElse(false);
         jreCertificates = JreCertificates.getInstance();
+        LOGGER.log(FINE, String.format("Loaded jre certificates: %s.", jreCertificates.getAliases()));
         wellKnowCertificates = SpecificPathCertificates.getSpecificPathCertificates(wellKnowPath);
+        LOGGER.log(FINE, String.format("Loaded well know certificates: %s.", wellKnowCertificates.getAliases()));
```
Comment (saragluna) [inline sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/KeyVaultKeyStore.java:137]: wellKnown

Comment (saragluna) [reply sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/KeyVaultKeyStore.java:137]: Shouldn't this be well known? Well know is not correct, right?

Comment (chenrujun) [reply sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/KeyVaultKeyStore.java:137]: New PR created: https://github.com/Azure/azure-sdk-for-java/pull/27426

**Pull request merged**